ner_telhub/widgets/styled_widgets.py

```diff
@@ -42,3 +45,63 @@ def changeStyle(self, style: Styles):
         self.style = style
         self.setStyleSheet(style.value)
 
+class Ner_toolbar():
+    
+    class Image():
+        def Set(self, image_path):
+            """
+            To add and image copy its relative path into the line 
+            Ner_toolbar.Image.Set(self, r'image path')
+            then use addAction enstead of addWidget
+            """
+            button = QPushButton()
+            button.setIcon(QIcon(image_path))
+            return button
+   
+    class addSideButton(QHBoxLayout): 
```

Comment:
A few things with this class:
- There is no need for it to be nested inside of Ner_toolbar. Instead, rename this class NERToolbar and delete the outer class Ner_toolbar
- Change the class to inherit from QToolbar instead of QHBoxLayout. If this causes issues let me know and we can find a workaround